autowrap/DeclResolver.py

```diff
@@ -144,6 +144,19 @@ def __init__(self, name, type_, decl):
         self.wrap_ignore = decl.annotations.get("wrap-ignore", False)
 
 
+default_namespace = ""
+
+def get_namespace(pxd, default_namespace):
+    filehandle = open(pxd)
+    fulltext = filehandle.read()
+    filehandle.close()
+    import re
+    match = re.search("cdef extern.*?namespace\s*\"([^\"]*)\"", fulltext)
+    if not match:
+        return default_namespace
+    else:
+        return match.group(1)
```

Comment:
not sure if this is the best way to get the namespace for a C++ class, this will just grab the first one. But I have not found another way to extract this from the Cython code itself